app/src/main/java/com/islamversity/reyan/BaseApp.kt

```diff
@@ -38,14 +38,22 @@ abstract class BaseApp : Application(), CoreComponentProvider {
 
     override fun onCreate() {
         super.onCreate()
+        if (BuildConfig.YandexAPIKey.isNotEmpty())
+            yandexMetricConfig()
+
+        Logger.init(listOf(LogcatLogger()))
+
+        AppFontSizeStore.init(this)
+    }
+
+    private fun yandexMetricConfig() {
```

Comment:
I would go for `initYandexMetric`
but not a big deal

Reply:
Consider it done.